docs/source/toy/neals_funnel.rst

```diff
@@ -0,0 +1,7 @@
+*****************************
+NEALS FUNNEL DISTRIBUTION
```

Comment:
Don't need all-caps here!
Also *s should have same length as the words below (or sphinx breaks)